crates/proto/src/proto.rs

```diff
@@ -395,6 +395,9 @@ messages!(
     (UpdateContext, Foreground),
     (SynchronizeContexts, Foreground),
     (SynchronizeContextsResponse, Foreground),
+    (LspExtSwitchSourceHeader, Background),
+    (LspExtSwitchSourceHeaderResponse, Background),
+    // Remote development
```

Comment:
```suggestion
```

Let's remove extra comments, as most of this list is oddly structured and there were no comments before?

Reply:
These comments were actually recently removed in 5f7881fc1e and I messed up the rebase :sweat_smile: